.changeset/blue-crews-hope.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/menu': minor
```

Comment:
note: none of these add up to a major for Menu - should we consolidate the changelogs and release this all under one big major for safety's sake?

Reply:
changesets will handle that, and put the change notes in the appropriate section in the changelog

Reply:
none of them are major though, so i'm saying to we batch them all and call it "major" even if we've identitifed these all individually as minor/patch


I do think this is a BIG change

Reply:
Ah you're right. This one should be major then